tests/unit/utilities/will-interrupt-process-test.js

```diff
@@ -160,16 +160,43 @@ describe('will interrupt process', function() {
           isTTY: true,
           on: td.function(),
           setRawMode: td.function(),
+          removeListener: td.function(),
         },
       });
 
       let windowsCtrlCTrap = td.matchers.isA(Function);
 
       willInterruptProcess.addHandler(cb);
-      td.verify(process.stdin.setRawMode(true));
       td.verify(process.stdin.on('data', windowsCtrlCTrap));
     });
 
+    it('adds and reverts rawMode on Windows', function() {
+      Object.defineProperty(process, 'platform', {
+        value: 'win',
+      });
+
+      Object.defineProperty(process, 'stdin', {
```

Comment:
I've tried to make it. But it messes up the pr. I will iterate on this in a separate pr.